tests/test_extractor.py

```diff
@@ -82,6 +87,11 @@ def py_files(filenames: Iterator[str]):
         for file_name in filenames:
             yield open(f'{SAMPLES_DIR}/{file_name}')
 
+    @staticmethod
+    def get_bytes_io_file(file_name) -> BytesIO:
+        with open(f'{SAMPLES_DIR}/{file_name}', 'br') as open_file:
```

Comment:
Use pathlib.Path and the / operator to concatinate paths. The `/` is not the path separator in all systems.